CHANGELOG

```diff
@@ -1,5 +1,9 @@
 devel
 -----
+
+* fix issue #4582: ui query editor now has the same behavior as java driver while
```

Comment:
This is a bit too general. We should describe the new behavior.
e.g.: fix issue #4582: UI query editor now supports usage of empty string as bind parameter value